cypress/e2e/requests/filter-requests-with-special-symbols.cy.js

```diff
@@ -0,0 +1,152 @@
+              status: { name: ITEM_STATUS_NAMES.AVAILABLE },
+              permanentLoanType: { id: itemData.loanTypeId },
+              materialType: { id: itemData.materialTypeId },
+            },
+          ],
+        });
+      })
+      .then((specialInstanceIds) => {
+        itemData.testInstanceIds = specialInstanceIds;
+        requestData.instanceId = specialInstanceIds.instanceId;
+        requestData.holdingsRecordId = specialInstanceIds.holdingIds[0].id;
+        requestData.itemId = specialInstanceIds.holdingIds[0].itemIds[0];
+      })
+      .then(() => {
+        cy.createTempUser([Permissions.requestsAll.gui])
+          .then((userProperties) => {
+            userData.username = userProperties.username;
+            userData.password = userProperties.password;
+            userData.userId = userProperties.userId;
+          })
+          .then(() => {
+            cy.wrap(true)
+              .then(() => {
+                requestData.requesterId = userData.userId;
+                requestData.pickupServicePointId = servicePoint.id;
+                requestData.patronComments = patronComments;
+              })
+              .then(() => {
+                cy.createItemRequestApi(requestData);
+              });
+
+            UserEdit.addServicePointsViaApi([servicePoint.id], userData.userId, servicePoint.id);
+            cy.login(userData.username, userData.password);
+          });
+      });
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    Requests.deleteRequestViaApi(requestData.id);
+    InventoryInstances.deleteInstanceAndHoldingRecordAndAllItemsViaApi(itemData.barcode);
+    UserEdit.changeServicePointPreferenceViaApi(userData.userId, [servicePoint.id]);
+    Users.deleteViaApi(userData.userId);
+    Location.deleteViaApiIncludingInstitutionCampusLibrary(
+      defaultLocation.institutionId,
+      defaultLocation.campusId,
+      defaultLocation.libraryId,
+      defaultLocation.id,
+    );
+    ServicePoints.deleteViaApi(servicePoint.id);
+  });
+
+  it(
+    'C374173 Verify filtering Requests by Tags with one special symbol (vega) (TaaS)',
+    { tags: ['extendedPath', 'vega'] },
+    () => {
+      cy.visit(TopMenu.requestsPath);
+      Requests.selectNotYetFilledRequest();
+      Requests.findCreatedRequest(itemData.instanceTitle);
+      Requests.selectFirstRequest(itemData.instanceTitle);
+      Requests.openTagsPane();
+      Requests.addNewTag(tag);
+      InteractorsTools.checkCalloutMessage('New tag created');
+
+      Requests.closePane('Tags');
+      Requests.closePane('Request Detail');
+      Requests.resetAllFilters();
+      Requests.filterRequestsByTag(tag);
+      Requests.selectFirstRequest(itemData.instanceTitle);
+      Requests.openTagsPane();
+      Requests.verifyAssignedTags(tag);
+
+      Requests.clearSelectedTag();
+      Requests.verifyNoResultMessage('Choose a filter or enter a search query to show results.');
+      Requests.filterRequestsByTag(tag);
```

Comment:
In step 6 and 8 you need to choose  a tag with two different ways
See enterTypeOfHeading and chooseTypeOfHeading as example